service/cloudfront/sign/sign_cookie.go

```diff
@@ -0,0 +1,141 @@
+package sign
+
+import (
+	"crypto/rsa"
+	"net/http"
+)
+
+//CookieSigner requirements are the same for the URL signer
+type CookieSigner struct {
+	keyID   string
+	privKey *rsa.PrivateKey
+}
+
+//CookieOptions optional settings for Cookie Definition, these can also be set manually
+type CookieOptions struct {
+	Path   string
+	Domain string
+	Secure bool
+}
+
+//Example:
+
+// func handler(w http.ResponseWriter, r *http.Request) {
```

Comment:
This example comment would be great inside of an `sign_cookie_example_test.go` file so it can be validated by the compiler, and could even add test output.